source/unified-test-format/unified-test-format.rst

```diff
@@ -382,6 +404,15 @@ The structure of this object is as follows:
   treat the comparison as not equal and skip the test. This includes errors that
   occur when fetching a single parameter using ``getParameter``.
 
+- ``loadBalanced``: Optional boolean. Specifies whether or not the tests can
+  be executed against a load-balanced cluster. If true, the relevant tests MUST
+  only be run against a load balancer. If false or omitted, the tests MUST only
+  be run against non-load balanced clusters.
```

Comment:
This echoes what Jeremy already said, but I'm concerned about the different handling of the `loadBalanced` flag compared to other options.

From what I gather, this means that any `runOnRequirements` entry implicitly has `loadBalanced: false` if it's not specified. However, what does this mean for tests where no requirements have been set? At the moment, these tests are always run, which according to the above should now change. If so, this must be clarified accordingly.

I understand that this may require a large number of test changes to exclude them from being run against load balanced setups, but unless there is good reason to not run a test (e.g. functionality not supported), I'd opt for running as many of our tests against a load balancer as well, removing the need for this different behaviour, the implicit `runOnRequirement`, or adding `loadBalanced: false` to all test files.

Reply:
I would be in favor of adding `loadBalanced: false` as needed vs. the alternative of changing how `runOnRequirement` is evaluated.

Having some implicit requirement check seems a like a can of worms, since we'd also then need to specify that it applies only to the top-level `runOnRequirements` and the corresponding test-level field. Consider if we only specified "if `runOnRequirements` is omitted, test runners must ensure the cluster is not load balanced." Then, even if a test-level `runOnRequirements` had `loadBalanced: true`, the test might still be skipped because there was no `runOnRequirements` at the top of the file. The same could apply vice versa.